File: a06.c
#include <stdio.h>
#include <math.h>

int main()
{
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    if (pow(b, 2) - 4 * a * c > 0)
    {
        int ans1 = (-b + sqrt(pow(b, 2) - (4 * a * c))) / (2 * a);
        int ans2 = (-b - sqrt(pow(b, 2) - (4 * a * c))) / (2 * a);
        printf("Two different roots x1=%d , x2=%d", ans1, ans2);
    }
    else if (pow(b, 2) - 4 * a * c == 0)
    {
        int ans = (-b + sqrt(pow(b, 2) - (4 * a * c))) / (2 * a);
        printf("Two same roots x=%d", ans);
    }
    else if (pow(b, 2) - 4 * a * c < 0)
    {
        printf("No real root");
    }

    return 0;
}

File: a06.py
import math
a, b, c = map(int, input().split(' '))

if b**2-(4*a*c) > 0:
    ans1 = int((-b+math.sqrt(b**2-(4*a*c)))/(2*a))
    ans2 = int((-b-math.sqrt(b**2-(4*a*c)))/(2*a))
    print("Two different roots x1=%d , x2=%d" % (ans1, ans2))
elif b**2-(4*a*c) == 0:
    ans = int((-b+math.sqrt(b**2-(4*a*c)))/(2*a))
    print("Two same roots x=%d" % ans)
else:
    print("No real root")
